src/main/java/dto/CityInfoDTO_OUT.java

```diff
@@ -27,6 +28,12 @@ public CityInfoDTO_OUT(Integer id, String zipCode, String city)
         this.zipCode = zipCode;
         this.city = city;
     }
+    
+    public CityInfoDTO_OUT(CityInfo city) {
+        this.id = city.getId();
+        this.city = city.getCity();
+        this.zipCode = city.getZipCode();
+    }
```

Comment:
This will create a merge conflict with PR #39 but it should be easy to fix

Reply:
Yeah, most likely. Good eye.